src/token.sw

```diff
@@ -25,3 +27,66 @@ pub fn force_transfer(amount: u64, asset_id: ContractId, contract_id: ContractId
         tr r3 r1 r2;
     }
 }
+
+// note: if tx format changes, the magic number "48" must be changed !
+// TransactionScript outputsCount has a 48 byte(6 words * 8 bytes) offset
+// Transaction Script: https://github.com/FuelLabs/fuel-specs/blob/master/specs/protocol/tx_format.md#transactionscript
+// Output types: https://github.com/FuelLabs/fuel-specs/blob/master/specs/protocol/tx_format.md#output
+// const OUTPUT_LENGTH_LOCATION = 48;
+// const OUTPUT_VARIABLE_TYPE = 4;
```

Comment:
Why don't you use these constants?

Reply:
Ah. I had disabled them while trying to pin down the source of the non-deterministic contract-ids issue.
They turned out to not be the cause, but I forgot to use them again. 
Will fix.